### PR TITLE
Replaces the bloodcrawl variable with a trait

### PR DESCRIPTION
**Pull request by SteelSlayer**
## What Does This PR Do
Removes the `bloodcrawl` variable from mobs, and replace it with a trait instead. 

In a majority of rounds, `bloodcrawl` is never even used. Having this variable be attached to ALL living mobs is wasteful.

No CL because this is a back-end change.

### CODE REVIEW

**Comment by SteelSlayer** [comment]
Had to remove the mimery stuff from this because I don't know how to handle mime's gaining the trait upon getting cloned or other body-transferring effects, still learning traits.

File: code/_globalvars/traits.dm [hunk 1]
```diff
@@ -5,7 +5,10 @@
 */
 GLOBAL_LIST_INIT(traits_by_type, list(
 	/mob = list(
-		"TRAIT_PACIFISM" = TRAIT_PACIFISM
+		"TRAIT_PACIFISM" = TRAIT_PACIFISM,
+		"TRAIT_WATERBREATH"	= TRAIT_WATERBREATH,
```
Comment (Fox-McCloud) [inline code/_globalvars/traits.dm:9]: Um.

File: code/__HELPERS/traits.dm [hunk 2]
```diff
@@ -62,7 +62,9 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 
 //mob traits
 #define TRAIT_PACIFISM			"pacifism"
-#define TRAIT_WATERBREATH       "waterbreathing"
+#define TRAIT_WATERBREATH		"waterbreathing"
```
Comment (Fox-McCloud) [inline code/__HELPERS/traits.dm:65]: uh!

Comment (SteelSlayer) [reply code/__HELPERS/traits.dm:65]: Uh yes I can explain, the area between `TRAIT_WATERBREATH` and `"waterbreathing"` were all spaces and it looked ugly, so I converted it to using tabs :smile:

File: code/_globalvars/traits.dm (hunk 1 above)
Comment (SteelSlayer) [reply code/_globalvars/traits.dm:9]: While I was adding bloodcrawl, I just decided to add waterbreath, because why not.

Comment (Fox-McCloud) [reply code/_globalvars/traits.dm:9]: but this isn't used anywhere!

Comment (SteelSlayer) [reply code/_globalvars/traits.dm:9]: If you mean the global list here, it's used in the same file a few times and in the 
`/datum/admins/proc/modify_traits(datum/D)` proc, which presumably allows admins to add/remove traits from mobs.

Comment (Fox-McCloud) [reply code/_globalvars/traits.dm:9]: nevermind, I was thinking something else *entirely*

**Pull request merged**